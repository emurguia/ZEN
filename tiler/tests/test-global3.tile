// This tests that global strings that have values assigned in one block can still be accessed in another block

string str;
int count;

init {
   tile(3,3);
   str = "test string";
   count = 0;
   sprint(str);
}

turn {
	sprint(str);
	count = count + 1;
	
}

end {

	if (count >= 1) {
		close();
		return 1;
	}
}