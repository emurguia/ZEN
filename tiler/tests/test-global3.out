test string
test string
The game window was successfully launched ...
The game window has been exited. Program exited.
Game over.